pkg/checks/external/main.go

```diff
@@ -1265,6 +1269,28 @@ func (ext *Checker) getPodClient() typedv1.PodInterface {
 	return ext.KubeClient.CoreV1().Pods(ext.Namespace)
 }
 
+// GenOwnerRef return the OwnerReference for objects(pod/cr) create by the khcheck checker
+func (ext *Checker) GenOwnerRef() ([]metav1.OwnerReference, error) {
+	khcheck, err := ext.getCheck()
+	if err != nil {
+		return nil, err
+	}
+
+	controller := true
+	blockOwnerDeletion := true
```

Comment:
Do you have the docs for this?  What does `blockOwnerDeletion` mean here?  Sorry for being lazy :-)

Reply:
https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/

blockOwnerDeletion is set to true, when we use the Foreground delete mode, the delete will be blocked. Although we use the Background delete mode, it does not exclude users from manually deleting the specified delete mode. Setting it to true here is relatively safe, and there will be no major problems.

Reply:
Thank you!